release/assets/macos_cli_installation.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -e
+
+USER_PROVIDED_PATH=$1
+
+main() {
+  cat << EOF
+   ______
+  / ____/___  ____    ( )_  _______
+ / /   / __ \/ __ \  / / / / / ___/
+/ /___/ /_/ / / / / / / /_/ / /
+\____/\____/_/ /_/_/ /\____/_/
+                /___/
+EOF
+
+  if [[ $USER_PROVIDED_PATH = "--help" ]]; then
+    print_help
+  fi
+  executable_path=${USER_PROVIDED_PATH:-'/usr/local/bin'}
+
+  echo "Configuring the Conjur CLI. This will take a few moments...."
+  create_symbolic_link $executable_path
+
+  conjur --help > /dev/null
+  echo "Successfully configured the Conjur CLI. To get started, run 'conjur --help'"
+}
+
+create_symbolic_link() {
+  # Creates symbolic link to the path of the user's choosing. Default location: '/usr/local/bin'
+  ln -s -f /$PWD/conjur $1
+}
+
+print_help() {
+  cat << EOF
+
+Conjur CLI Installation script
+
+Installs and configures the Conjur CLI on macOS environments. Upon completion of this configuration script, you will be able to access the Conjur CLI from anywhere on your machine.
+A symbolic link will be created between the CLI executable and, by default, '/usr/local/bin' but this file location can be overwritten. To do so, provide a parameter with the desired path.
+For example ./macos_cli_installation.sh <new-path-location>
+
```

Comment:
Installs and configures the Conjur CLI on macOS environments. 
Upon completion of this configuration script, you will be able to access the Conjur CLI from anywhere on your machine.
The installation creates a symbolic link between the CLI executable and the ???? file path. By default, this path is '/usr/local/bin'. To overwrite this path,  configure the ??? parameter. 
For example ./macos_cli_installation.sh <new-path-location>


@sigalsax  WDYT? Can you fill in the "???" ? It's the path to what?

Reply:
Installs and configures the Conjur CLI on macOS environments.
Upon completion of this setup script, you will be able to access the Conjur CLI from anywhere on your machine.
As a result, a symbolic link between the CLI executable and the ???? file path. By default, this path is '/usr/local/bin'. To overwrite this path, configure the ??? parameter.
For example ./macos_cli_installation.sh

Reply:
??? is a directory path